e2e/testcases/birth/1-birth-event-declaration.spec.ts

```diff
@@ -1,54 +1,560 @@
+          page.getByRole('button', { name: 'Save & Exit' })
+        ).toBeVisible()
+        await page.locator('#eventToggleMenuToggleButton').click()
+        await expect(
+          page.getByRole('button', { name: 'Delete declaration' })
+        ).toBeVisible()
+      })
 
-  test('1.1. Navigate to the birth event declaration page', async ({
-    page
-  }) => {
-    await page.click('#header_new_event')
-    await page.waitForSelector('#continue')
+      test.skip('1.8.2 Validate Supporting Document block', async () => {})
+
+      test('1.8.3 click continue', async () => {
+        await page.getByRole('button', { name: 'Continue' }).click()
+
+        /*
+         * Expected result: should navigate to "Review" page
+         */
+        await expect(
+          page
+            .getByRole('button', { name: 'Send for review' })
+            .or(page.getByRole('button', { name: 'Register' }))
+        ).toBeVisible()
+      })
+    })
+
+    test.describe('1.9 Validate "Save & Exit" Button  ', async () => {
+      test('1.9.1 Click the "Save & Exit" button from any page', async () => {
+        await page.getByRole('button', { name: 'Save & Exit' }).click()
+
+        /*
+         * Expected result: should open modal with:
+         * - Title: Save & Exit
+         * - Helper text: All inputted data will be kept secure for future editing.
+         *                Are you ready to save any changes to this declaration form?
+         * - Cancel Button
+         * - Confirm Button
+         */
+        await expect(
+          page.getByRole('heading', { name: 'Save & exit?' })
+        ).toBeVisible()
+        await expect(
+          page.getByText(
+            'All inputted data will be kept secure for future editing. Are you ready to save any changes to this declaration form?'
+          )
+        ).toBeVisible()
+        await expect(page.getByRole('button', { name: 'Cancel' })).toBeVisible()
+        await expect(
+          page.getByRole('button', { name: 'Confirm' })
+        ).toBeVisible()
+      })
+
+      test('1.9.2 Click Cancel', async () => {
+        await page.getByRole('button', { name: 'Cancel' }).click()
+
+        /*
+         * Expected result: should close the modal
+         */
+        await expect(
+          page.getByRole('heading', { name: 'Save & exit?' })
+        ).toBeHidden()
+      })
+
+      test('1.9.3 Click Confirm', async () => {
+        await page.getByRole('button', { name: 'Save & Exit' }).click()
+        await page.getByRole('button', { name: 'Confirm' }).click()
+
+        /*
+         * Expected result: should
+         * - be navigated to "in-progress" tab
+         * - find the declared birth event record on this page list with saved data
+         */
+        await expect(page.locator('#content-name')).toHaveText('In progress')
+        await expect(page.getByText('0 seconds ago')).toBeVisible()
+      })
+    })
   })
+  test.describe('1.10 Validate "Exit" Button  ', async () => {
```

Comment:
```suggestion
  test.describe('1.10 Validate "Exit" Button', async () => {
```

A couple of trailing whitespaces in the descriptions